app/models/shipit/stack.rb

```diff
@@ -528,8 +528,25 @@ def sync_github
       GithubSyncJob.perform_later(stack_id: id)
     end
 
+    alias links_orig links
```

Comment:
Rather than aliasing the method, you can remove it from the delegation list, and call `cached_deploy_spec.links`.

Also note that this `delegate` has a `allow_nil: true`, so you have to handle `cached_deploy_spec` being nil.

Reply:
Addressed by https://github.com/Shopify/shipit-engine/pull/1018/commits/c2ec62ac53841a7e3d2f32e1530916a94428e964